GameLamp/src/Core/Log.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <spdlog/spdlog.h>
+#include <memory>
+
+namespace Lamp 
+{
+	class Logger 
+	{
+	public:
+		static void Init();
+
+		inline static std::shared_ptr<spdlog::logger>& GetCoreLogger() { return m_CoreLogger; }
+		inline static std::shared_ptr<spdlog::logger>& GetClientLogger() { return m_ClientLogger; }
+	private:
+		static std::shared_ptr<spdlog::logger> m_CoreLogger;
+		static std::shared_ptr<spdlog::logger> m_ClientLogger;
+	};
+}
+
+#ifdef NDEBUG
```

Comment:
Different log levels have to be enabled / disabled depending on different build configuration (Debug, Release, Shipping)
Debug - all logs are enabled
Release - error and critical
Shipping - critical only or disable all logs